[docs] #249: Add the support channels (#250)
hyperledger/iroha-2-docs: src/guide/support.md

# Receive support

From time to time, you may have questions about Iroha that you would like to discuss in detail with others.

> There are three ways to quickly get in touch with our community: Telegram, Discord and GitHub.

Reviewer: ```suggestion
There are three ways to quickly get in touch with our community: Telegram, Discord, and GitHub.
```

A large part of the community currently uses [Telegram](https://t.me/hyperledgeriroha) for communication.
The Hyperledger part of the team prefers [Discord](https://discord.gg/hyperledger), with two dedicated channels: `iroha` and `iroha-2-contributors`.

> The Discord and Telegram channels are synchronised, and users of both channels see your messages.

Reviewer 2: ```suggestion
The Discord and Telegram channels are synchronized, so users of both media see your messages.
```

> Finally, you can [create](https://github.com/hyperledger/iroha/issues/new/choose) a GitHub issue, whether it's a request to update documentation, a suggestion for the core team, or a bug you have found.

Reviewer 2: ```suggestion
Finally, you can create a [GitHub issue] (https://github.com/hyperledger/iroha/issues/new/choose), whether it's a request to update documentation, a suggestion for the core team, or a bug you have found.
```

Reviewer: or maybe?

```suggestion
Finally, you can [create a GitHub issue] (https://github.com/hyperledger/iroha/issues/new/choose), whether it's a request to update documentation, a suggestion for the core team, or a bug you have found.
```

Reviewer 2: That looks much better
